src/connection.cc

```diff
@@ -216,7 +220,11 @@ NAN_METHOD(Connection::Ftype) {
 
   PGresult* res = self->lastResult;
 
-  int colName = PQftype(res, info[0]->Int32Value());
+  #if defined(V8_MAJOR_VERSION) && V8_MAJOR_VERSION >= 7
```

Comment:
Use `Nan::To<>` instead an `ifdef` and v8 apis (see https://github.com/nodejs/nan/blob/master/doc/converters.md)

Reply:
Something like this ?
`info[0]->ToInt32(Nan::GetCurrentContext()).ToLocalChecked()->Value())`

Reply:
more like this: `Nan::To<int64_t>(info[0]).FromJust()` see e.g. https://github.com/nodejs/nan/blob/master/test/cpp/converters.cpp

Reply:
Oh I see, sorry I am really struggling with this :p
Last time I touched Cpp it was 10 years ago at least ^^'

Thank you for taking the time to review my code ^^